backend/src/main/java/quarano/core/MessageConfiguration.java

```diff
@@ -0,0 +1,43 @@
+package quarano.core;
+
+import org.springframework.context.MessageSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.support.MessageSourceAccessor;
+import org.springframework.context.support.ReloadableResourceBundleMessageSource;
+import org.springframework.validation.Validator;
+import org.springframework.validation.beanvalidation.LocalValidatorFactoryBean;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+/**
+ * @author Felix Schultze
+ */
+@Configuration
+public class MessageConfiguration implements WebMvcConfigurer {
```

Comment:
Why do we need this explicit configuration? Everything but the `MessageSourceAccessor` that was previously already declared in `Quarano` is exactly what Boot configures by default.

Reply:
Its strange. We need to return the Validator to resolve the error messages. If I just return an autowired LocalValidatorFactoryBean, it's not working. I have to set the messagesource on the LocalValidatorFactoryBean manually. So I decieded to define it completely

Reply:
JSR 303 validation has already worked before in general, that's why I was puzzled.